src/clase/Jugador.java

```diff
@@ -0,0 +1,48 @@
+package clase;
+public class Jugador {
+    private String nombre;
+    private int edad;
+    private int partidasGanadas;
+    private int rachaActual;
+
+    // Constructor
+    public Jugador(String nombre, int edad) {
+        this.nombre = nombre;
+        this.edad = edad;
+        this.partidasGanadas = 0;
+        this.rachaActual = 0;
+    }
+
+    // Getters y Setters
+    public String getNombre() {
+        return nombre;
+    }
+
+    public void setNombre(String nombre) {
+        this.nombre = nombre;
+    }
+
+    public int getEdad() {
+        return edad;
+    }
+
+    public void setEdad(int edad) {
+        this.edad = edad;
+    }
+
+    public int getPartidasGanadas() {
+        return partidasGanadas;
+    }
+
+    public void setPartidasGanadas(int partidasGanadas) {
+        this.partidasGanadas = partidasGanadas;
+    }
+
+    public int getRachaActual() {
+        return rachaActual;
+    }
+
+    public void setRachaActual(int rachaActual) {
+        this.rachaActual = rachaActual;
+    }
+}
```

Comment:
Al final de cada archivo agrega una linea vacia, sino GitHub te lo marca en rojo
```suggestion
}

```
Asi:
![Screenshot 2025-05-05 at 10 30 31 PM](https://github.com/user-attachments/assets/c72a3528-deb1-4aac-855d-87ee512c0a69)